website/server/models/group.js

```diff
@@ -611,9 +611,9 @@ schema.methods.sendChat = function sendChat (options = {}) {
   };
 
   User
-    .update(query, lastSeenUpdateRemoveOld, { multi: true })
+    .updateMany(query, lastSeenUpdateRemoveOld, { multi: true })
```

Comment:
`multi: true` is redundant with the `updateMany` function, can remove

Reply:
done!